metrics/web/test/common/presentation/injector/widget/injection_container_test.dart

```diff
@@ -21,10 +24,64 @@ void main() {
       await tester.pumpAndSettle();
     }
 
+    testWidgets(
+      "throws an AssertionError if the given metrics config is null",
+      (tester) async {
+        await tester.pumpWidget(const InjectionContainerTestbed(
+          metricsConfig: null,
+        ));
+
+        expect(tester.takeException(), isAssertionError);
+      },
+    );
+
+    testWidgets(
+      "creates a new instance with the given metrics config",
+      (tester) async {
+        const metricsConfig = MetricsConfigStub(sentryEnvironment: 'test');
+
+        await tester.pumpWidget(const InjectionContainerTestbed(
+          metricsConfig: metricsConfig,
+        ));
+        final finder = find.byType(InjectionContainer);
+        final container = tester.widget<InjectionContainer>(finder);
```

Comment:
```suggestion
        final injectionContainer = tester.widget<InjectionContainer>(finder);
```
will that improve reading expect()?